canonical-web-and-design/maas-ui
# Add zone details header component

## Done

- Organise zone details and zone list into their own directories
- Add Zone details header component
- Add id to link in zone list table
- Add tests for header

## QA

- run `yarn start` and go to http://0.0.0.0:8400/MAAS/r/zone/3 - check you can see the zone name and button
- go to http://0.0.0.0:8400/MAAS/r/zone/334 check you see not found message

![Screenshot 2021-06-10 at 14 50 07](https://user-images.githubusercontent.com/58959073/121536896-5e3d9300-c9fb-11eb-8479-5a6df3d4b256.png)


![Screenshot 2021-06-10 at 14 50 22](https://user-images.githubusercontent.com/58959073/121536829-51b93a80-c9fb-11eb-9e5d-df725c87c1aa.png)
